docs/index.rst

```diff
@@ -28,6 +28,15 @@ Welcome to Concept to Clinic's documentation!
    code-of-conduct-link
    contributor-license-agreement-link
 
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Code Documentation
+
+   apidoc_interface/interface_doc
```

Comment:
This `apidoc` entries should not be here any more since (https://github.com/concept-to-clinic/concept-to-clinic/commit/ef5395f8ef341567093b22c94e6c596fae35584a).

Reply:
updating these changes 👍